setup.py

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from setuptools import setup
+from setuptools import find_packages, setup
+
 from djangocms_video import __version__
 
+
+REQUIREMENTS = [
+    'django-cms>=3.3.1',
```

Comment:
@czpython what would be the lowest requirement I could add here? 1.1 should have high compatibility with Django 1.6+ and CMS ???. 2.0 will drop support for Django < 1.8 (similar to djangocms-audio)